tests/e2e/specs/api/CloneGitRepoAPI.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { OpenshiftHelper } from '../../utils/OpenshiftHelper';
+import { expect } from 'chai';
+import { ShellString } from 'shelljs';
+import { GitUtil } from '../../utils/vsc/GitUtil';
+
+suite(`Empty Workspace export repository tests`, async function (): Promise<void> {
```

Comment:
Suite info doesn't completely comply with test file name and test scenario.

It is currently `Test cloning of repo "https://github.com/crw-qe/web-nodejs-sample" into empty workspace`.